core/xml.js

```diff
@@ -301,6 +301,25 @@ Blockly.Xml.domToWorkspace = function(xml, workspace) {
   if (workspace.setResizesEnabled) {
     workspace.setResizesEnabled(false);
   }
+  
+  // find last block on workspace to add new blocks at the bottom 
+  var blocks = workspace.getAllBlocks();
+  var bottomY =0; // container for the y of the bottom block
+  // sort the blocks in reverse order
+  if (blocks.length > 1) {
+	 blocks.sort(function(a, b) { // sort the block acording to their position and height
```

Comment:
Indeed, thanks.
If I want to align the block, as suggested by AnmAtAnm, I will also need the x position, so a loop through the blocks will be better.